src/lib/crypto.cpp

```diff
@@ -198,6 +198,10 @@ key_material_equal(const pgp_key_material_t *key1, const pgp_key_material_t *key
 rnp_result_t
 validate_pgp_key_material(const pgp_key_material_t *material, rng_t *rng)
 {
+#ifdef FUZZERS_ENABLED
```

Comment:
Increasing `fuzz-seconds` in `cifuzz.yml` didn't help?

Reply:
Yeah, it affects only run of all fuzzing attempts, not the single data block processing. Still decided to leave fuzz-seconds value higher for more confidence that there are no fuzzing blockers.